pycbc/noise/reproduceable.py

```diff
@@ -81,7 +81,7 @@ def normal(start, end, sample_rate=16384, seed=0):
     sv = RandomState(seed).randint(-2**50, 2**50)
     data = numpy.concatenate([block(i + sv, sample_rate)
                               for i in numpy.arange(s, e + 1, 1)])
```

Comment:
Does this need to be `e+2`?

Reply:
@spxiwh Why? e + 1 should have an end time that is beyond the requested end. It's only the block at 'e' that might end earlier due to rounding. If this were a problem, we would have seen errors before when generated lots of fake noise.

Reply:
Hmm, yeah, think I miscounted. `e+1` does seem right.